### PR TITLE
Clear session storage after removing cookie

### LINKED ISSUE #16: Set frontend cookies to expire from backend
Hey @alex-oleshkevich !

Isn't it possible to clear the frontend session cookie for most browsers?

By setting it to an empty value and an expiry date that has already passed, most browsers would know to clear it.

Even if the browser didn't clear it, by setting it to an empty value it is no longer valid.
```Set-Cookie: token=deleted; path=/; expires=Thu, 01 Jan 1970 00:00:00 GMT```

We could have this take place as part of `session.clear()`. What do you think?

### PR DESCRIPTION
**Pull request by alex-oleshkevich**
Cleanup storage after cookie removal.

Resolves #16

### CODE REVIEW

**Comment by alex-oleshkevich** [comment]
@RamiAwar you may want to review it.

**Review by RamiAwar — APPROVED** [review]
lgtm

File: starsessions/middleware.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ async def send_wrapper(message: Message) -> None:
                             self.security_flags,
                         )
                         headers.append("Set-Cookie", header_value)
+                        await self.backend.remove(scope['session'].session_id)
```
Comment (RamiAwar) [inline starsessions/middleware.py:83]: I just noticed that this was missing :D Was about to make this change

**Comment by RamiAwar** [comment]
It's worth making a minor release after those changes.

**Pull request merged**